pages.de/common/arping.md

```diff
@@ -0,0 +1,29 @@
+# arping
+
+> Mit dem ARP Protokoll Hosts in einem Netzwerk entdecken und untersuchen.
+> Nützlich für die Entdeckung MAC Addressen.
+> Weitere Informationen: <https://github.com/ThomasHabets/arping>.
+
+- Mit ARP Requests Paketen einen Host pingen:
+
+`arping {{host_adresse}`
+
+- Einen Host auf einem spezifizierten Interface pingen:
+
+`arping -I {{interface}} {{host_adresse}}`
+
+- Einen Host pungen und nach der ersten Antwort aufhören:
+
+`arping -f {{host_adresse}}`
+
+- Einen Host für eine bestimmte Anzahl pingen:
```

Comment:
Worst phrasing ever. I asked ChatGPT and even it couldn't help me

<img width="611" alt="Pixel: Wie sagt man 'do something a specific number of times' auf Deutsch (im Imperativ)? ChatGPT: Um im Imperativ auszudrücken, dass jemand etwas eine bestimmte Anzahl von Malen tun soll, kann man im Deutschen folgende Konstruktion verwenden: 'Tu es (Anzahl) Mal!' Zum Beispiel: 'Schreib es fünf Mal auf!' 'Sag es dreimal laut!' 'Lies den Text zweimal durch!' 'Übe den Trick zehn Mal!' In diesem Fall wird das Verb im Infinitiv benutzt und die Anzahl der Wiederholungen wird mit dem entsprechenden Zahlwort angegeben. Pixel: wie formuliere ich es mit einer phrase, die eine nicht bekannte Anzahl ausdrückt (analog zu 'a specific number of times') ChatGPT: Um auszudrücken, dass jemand etwas eine nicht bekannte Anzahl von Malen tun soll, kann man im Imperativ im Deutschen folgende Konstruktion verwenden: 'Tu es mehrmals!' Zum Beispiel: 'Übe den Trick mehrmals, bis du ihn beherrschst!' 'Lies den Text mehrmals, um ihn besser zu verstehen!' 'Sag es mehrmals laut, damit es im Gedächtnis bleibt!' 'Schreib es mehrmals auf, um es auswendig zu lernen!' In diesem Fall wird das Verb im Infinitiv benutzt und die Phrase 'mehrmals' gibt an, dass die Handlung mehr als einmal ausgeführt werden soll, aber keine spezifische Anzahl festgelegt wird." src="https://user-images.githubusercontent.com/35269695/229135597-524fa3ac-c962-4359-ba4a-c58724fd11be.png">


```suggestion
- Pinge einen Host für eine bestimmte Anzahl:
```